src/Microsoft.AspNet.Mvc.Core/ActionResults/ObjectResult.cs

```diff
@@ -38,6 +38,18 @@ public ObjectResult(object value)
         /// </summary>
         public int? StatusCode { get; set; }
 
+        /// <summary>
+        /// Gets or sets the flag to exclude formatters that match only on the object type.
```

Comment:
Is this accurate? it does not exclude formatters but it skips matching based on object type